sphinx/environment/__init__.py

```diff
@@ -214,6 +211,14 @@ def __init__(self, srcdir, doctreedir, config):
         # attributes of "any" cross references
         self.ref_context = {}
 
+        self.init_managers()
+
+    def init_managers(self):
```

Comment:
I could not found good naming for these modules.
I think "manager" is not a best name. So I would be glad if any advice.